api/client/samples/anomaly_detection/sample_anomaly_detection.ipynb

```diff
@@ -0,0 +1,787 @@
+{
```

Comment:
<p>use get_descendant_regions here to ensure you get the right region level. </p><p>districts_list = [region['id'] for region in gclient.get_descendant_regions(salta_region_id, 5)] #districts are region level 5 in Gro's ontology</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>